config/routes.rb

```diff
@@ -9,18 +9,10 @@
 
   resources :users, only: [:edit, :update] do
     get 'edit-account', to: 'users#account'
-    get "portfolio", to: "users/portfolios#show"
-    get "portfolio/new", to: "users/portfolios#new"
-    get "portfolio/edit", to: "users/portfolios#edit"
-    put "portfolio", to: "users/portfolios#update"
-    post "portfolio", to: "users/portfolios#create"
-    delete "portfolio", to: "users/portfolios#destroy"
-    get "portfolio/delete", to: "users/portfolios#delete"
-    get "project/new", to: "users/projects#new"
-    post "project", to: "users/projects#create"
-    delete "project", to: "users/projects#destroy"
-    get "project/edit", to: "users/projects#edit"
-    put "project", to: "users/projects#update"
+    resource :portfolio, :controller => "users/portfolios", only: [:show, :new, :create, :edit, :update, :destroy] do
```

Comment:
@drod1000 Looks great!!! I see that you were able to figure out to have the controller under `users/portfolios` instead of `portfolios`.

I think this line can further be refactored: `only: [:show, :new, :create, :edit, :update, :destroy]` is redundant as using `resource` (singular) already takes care of generating all routes except `index`. I attached screenshot confirming the routes are the same without `only: [:show, :new, :create, :edit, :update, :destroy]`.

Attached screenshot terminal window on the left is `rake routes` output of PR #84 code, terminal window on the right is `rake routes` output without `only: [:show, :new, :create, :edit, :update, :destroy]`

<img width="1430" alt="screen shot 2017-03-28 at 8 41 12 pm" src="https://cloud.githubusercontent.com/assets/18202283/24436163/37050156-13f7-11e7-9df8-590fb99bc6fc.png">

So in conclusion I think this line should be the following:
`resource :portfolio, :controller => "users/portfolios" do`. With this shorter code still `rspec` output is the same as for PR #84.

@jdconrad89 @mollybrown What do you think?